src/frontend/MiljøProvider.tsx

```diff
@@ -12,10 +12,10 @@ import { FeatureTogglesProvider } from './context/FeatureToggleContext';
 import { InnloggetProvider } from './context/InnloggetContext';
 import { LastRessurserProvider } from './context/LastRessurserContext';
 import { SanityProvider } from './context/SanityContext';
-import { useSpråk } from './context/SpråkContext';
+import { useSpråkContext } from './context/SpråkContext';
 
 const MiljøProvider: React.FC<PropsWithChildren> = ({ children }) => {
-    const { valgtLocale } = useSpråk();
+    const { valgtLocale } = useSpråkContext();
```

Comment:
De fleste av filene i denne PRen har endringer som denne, der jeg har renamet hooken til en contekst slik at det er tydelig at man henter noe fra en context. Disse endringene ligger i egne commits for å gjøre det ryddigere å se over koden ☺️